src/modules/iotjs_module_http_parser.c

```diff
@@ -131,19 +131,20 @@ static void iotjs_http_parserwrap_flush(
       iotjs_jval_get_property(jobj, IOTJS_MAGIC_STRING_ONHEADERS);
   IOTJS_ASSERT(jerry_value_is_function(func));
 
-  iotjs_jargs_t argv = iotjs_jargs_create(2);
   jerry_value_t jheader = iotjs_http_parserwrap_make_header(http_parserwrap);
-  iotjs_jargs_append_jval(&argv, jheader);
-  jerry_release_value(jheader);
+  size_t argc = 1;
+  jerry_value_t argv[2] = { jheader, jerry_create_null() };
```

Comment:
There seem two approaches to this.

a) explicitly adding `jerry_null` into the `argv`before giving it to `iotjs_invoke_callback`.
b) verifying the given `argv` inside `iotjs_invoke_callback` and auto-filling `jerry_null` if any item is `null`.

What would be better?

Reply:
There could be an init function, but for such a small amount of values, its cost is higher than its worth.

The auto fill will probably not work, because argv has jerry_value_t, and its values are random, not pre-defined, since it is a local value allocated on the stack.

Reply:
In this case, the values can be pre-defined as 0 like `jerry_value_t argv[2] = { jheader };`. For all zero items, `jerry_value_t argv[2] = {};`. It may be simpler than the current way.

Reply:
`jerry_value_t argv[2] = {};` does not guarantee the values will be `0`. It is allocated on the stack so it may contain non-zero values too.

Reply:
At least, it's allowed in C++, as I remember, but not sure for C standard. If not, we could do `jerry_value_t argv[2] = {0};` for initializing all elements, IMO.